src/styles/blocks/reviws.scss

```diff
@@ -0,0 +1,40 @@
+.reviews {
+  display: flex;
+  margin-bottom: 24px;
+  justify-content: space-between;
+
+  &__stars {
+    display: flex;
+
+    &-stars {
+      width: 16px;
+      height: 16px;
+      left: 80px;
+      border-radius: 0.5px;
+      margin-right: 4px;
+      background-image: url('./images/star.svg');
+      background-repeat: no-repeat;
+      background-position: center center;
+    }
+
+    &-stars:nth-child(-n + 4) {
+      background-image: url('./images/star-active.svg');
+    }
+  }
+
+  &__reviews {
+    color: $main-text-color;
+    width: 53px;
+    height: 16px;
+    display: flex;
+    justify-content: right;
+    align-items: end;
+    font-family: $main-font-famaly;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Ensure that the variable is correctly defined and imported.